src/foraging_gui/TransferToNWB.py

```diff
@@ -489,8 +493,6 @@ class obj:
     logger.setLevel(logging.DEBUG)
     logger.addHandler(logging.StreamHandler())
     
-    # bonsai_to_nwb(R'F:\Data_for_ingestion\Foraging_behavior\Bonsai\AIND-447-G1\668546\668546_2023-09-19.json')
-    
-    bonsai_to_nwb(R'F:\Data_for_ingestion\Foraging_behavior\Bonsai\AIND-447-1-D\707254\707254_2024-02-16_13-18-41\TrainingFolder\707254_2024-02-16_13-18-41.json')
+    bonsai_to_nwb(R'F:\Data_for_ingestion\Foraging_behavior\Bonsai\AIND-447-G1\668546\668546_2023-09-19.json')
```

Comment:
Whats the logic of hard-coding `AIND-447-G1` ? G1 is an outdated computer name. This code will also be run on computers outside of 447 (ephys room, for example)

Reply:
This is just for local testing on my PC. When the file is plugged into a pipeline, the user should import and call `bonsai_to_nwb`.